contents/data_vis_makie_recipe.md

```diff
@@ -0,0 +1,96 @@
+## A Makie recipe for a DataFrame {#sec:recipe_df}
+
+Unlike other libraries that already support a wide set of input formats via recipes, i.e. `Plots.jl`, in `Makie.jl` most of the time we need to pass the raw data to our functions.
+However, if we want to define our own `recipe` in Makie we could also do it.
+As an example we will code a small **full recipe** for a `DataFrame`. Please refer to the [documentation](https://makie.juliaplots.org/stable/documentation/recipes/) for more details.
+
+A Makie `recipe` consist of two parts, a plot `type` name defined via `@recipe` and a custom `plot!`(`Makie.plot!`) which creates the actual plot via plotting functions already defined.
+
+```jl
+s = """
+    @recipe(DfPlot, df) do scene
+        Attributes(
+            x = :A,
+            y = :B,
+            c = :C,
+            color = :red,
+            colormap = :plasma,
+            markersize = 20,
+            marker = :rect
+        )
+    end
+    """
+sc(s)
+```
+
+with this we will have `dfplot` and `dfplot!` conveniently defined for us.
```

Comment:
```suggestion
with this we will have `dfplot` and `dfplot!` conveniently defined for us.
With this, we can conveniently plot `DataFrame`s which contains columns named, `x`, `y`, `z`.
```

Reply:
"With this" is duplicated as starting point for two times in a row on your suggestion @rikhuijzer.

Reply:
Fixed